### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by andrewxhill**
Added terms and licenses content

### CODE REVIEW

File: catalog/source/index.rst [hunk 1]
```diff
@@ -24,3 +24,10 @@ Indices and tables
 * :ref:`modindex`
 * :ref:`search`
 
+
+Terms and Licenses 
+==================
+
+The Data Observatory is a collection of various sources of data with varying licenses. We have worked hard to find you data that will work for the broadest set of use-cases. Please still review the below terms for any dataset you use and please respect the rights of the owners of each dataset. Please use the following link to review third-party data sources that are used in the Data Observatory, along with links to the terms governing the use of each.
+
+http://docs.cartodb.com/cartodb-platform/dataobservatory/license/
```
Comment (talos) [inline catalog/source/index.rst]: @andrewxhill Could you make this an RST external link a la

    External hyperlinks, like `Python <http://www.python.org/>`_.

From (http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets)

Comment (andrewxhill) [inline catalog/source/index.rst]: updated

**Comment by talos** [comment]
@andrewxhill looks good.  Only note is that the link is currently dead.

**Comment by andrewxhill** [comment]
yep that's fine for now!...

On Tue, May 17, 2016 at 11:47 AM, john krauss <notifications@github.com>
wrote:

> @andrewxhill <https://github.com/andrewxhill> looks good. Only note is
> that the link is currently dead.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/bigmetadata/pull/29#issuecomment-219761386>
>



-- 

andrewxhill <http://twitter.com/andrewxhill>
http://cartodb.com

**Pull request merged**